### DISCUSSION TITLE
Proposal: hide completed shows gray-checks

### DISCUSSION THREAD

**Discussion by freyley** [iSoron/uhabits#808] [Feature Requests]
Gray-checks are things that I don't have to do, but I still might want to do. For example, if I've got an exercise habit that's 3x/week, I might still want to do a 4th this week. 

The "hide completed" mode is super useful for showing me a todo list of what's remaining, but I can't see gray-checks, so I have to unhide to be able to complete them. The "I finished it and it vanished" feeling is really nice, but I don't get it in the show-all mode. 

Could hide completed show the gray-checks? Or could there be an additional hide, maybe "hide all complete" which shows what hide completed shows now? Or a new hide, to show xs and gray-checks, but I'm not sure what that would be called.

It could also sort them to the bottom of the list, or that could be an extra sort? But that seems entirely optional and an extra feature.

**Comment by naught101** [comment]
I think that this would be really good in combination with sending completed habits to the bottom of the list: https://github.com/iSoron/uhabits/discussions/827
